### PR TITLE
Feat: add option to ommit variants on SEOPLP

### PR DESCRIPTION
**Pull request by IncognitaDev**

### CODE REVIEW

**Comment by guitavano** [comment]
Can you put the motivation behind this suggestion?

**Comment by IncognitaDev** [comment]
I'am not fully into the motivation, but it has been requested multiple times by SEO agencies of our costumers, Like Torra and now Trisex.  

Its ralated with Google Rich Results, before, PDP has 10 products detected (https://search.google.com/test/rich-results/result?id=_6j9U24yHLNbrwIkC_LcUw), now with this change (in our custom seo section) its detected only one item, that its the required result (https://search.google.com/test/rich-results/result?id=bfaBuXvHxtQzQmV_XTY0rA).

File: commerce/sections/Seo/SeoPDP.tsx [hunk 1]
```diff
@@ -4,9 +4,10 @@ import { canonicalFromBreadcrumblist } from "../../utils/canonical.ts";
 
 export type Props = {
   jsonLD: ProductDetailsPage | null;
+  ommitVariants?: boolean;
 } & Partial<Omit<SeoProps, "jsonLDs">>;
 
-function Section({ jsonLD, ...props }: Props) {
+function Section({ jsonLD, ommitVariants, ...props }: Props) {
```
Comment (mcandeia) [inline commerce/sections/Seo/SeoPDP.tsx:10]:
```suggestion
function Section({ jsonLD,omitVariants, ...props }: Props) {
```

File: commerce/sections/Seo/SeoPDP.tsx [hunk 2]
```diff
@@ -4,9 +4,10 @@ import { canonicalFromBreadcrumblist } from "../../utils/canonical.ts";
 
 export type Props = {
   jsonLD: ProductDetailsPage | null;
+  ommitVariants?: boolean;
```
Comment (mcandeia) [inline commerce/sections/Seo/SeoPDP.tsx:7]:
```suggestion
 omitVariants?: boolean;
```

File: commerce/sections/Seo/SeoPDP.tsx [hunk 3]
```diff
@@ -17,6 +18,10 @@ function Section({ jsonLD, ...props }: Props) {
     : undefined;
   const noIndexing = !jsonLD;
 
+  if (ommitVariants && jsonLD?.product.isVariantOf?.hasVariant) {
```
Comment (mcandeia) [inline commerce/sections/Seo/SeoPDP.tsx:21]:
```suggestion
  if (omitVariants && jsonLD?.product.isVariantOf?.hasVariant) {
```

**Review by mcandeia — CHANGES_REQUESTED** [review]

File: commerce/sections/Seo/SeoPDP.tsx [hunk 4]
```diff
@@ -4,9 +4,10 @@ import { canonicalFromBreadcrumblist } from "../../utils/canonical.ts";
 
 export type Props = {
   jsonLD: ProductDetailsPage | null;
+ omitVariants?: boolean;
 } & Partial<Omit<SeoProps, "jsonLDs">>;
 
-function Section({ jsonLD, ...props }: Props) {
+function Section({ jsonLD,omitVariants, ...props }: Props) {
```
Comment (mcandeia) [inline commerce/sections/Seo/SeoPDP.tsx:10]:
```suggestion
function Section({ jsonLD, omitVariants, ...props }: Props) {
```

File: commerce/sections/Seo/SeoPDP.tsx [hunk 5]
```diff
@@ -4,9 +4,10 @@ import { canonicalFromBreadcrumblist } from "../../utils/canonical.ts";
 
 export type Props = {
   jsonLD: ProductDetailsPage | null;
+ omitVariants?: boolean;
```
Comment (mcandeia) [inline commerce/sections/Seo/SeoPDP.tsx:7]:
```suggestion
   omitVariants?: boolean;
```

**Review by mcandeia — APPROVED** [review]
Fix formatting

**Pull request merged**